gabarit/template_nlp/nlp_project/tests/test_utils.py

```diff
@@ -244,6 +244,49 @@ def test10_find_folder_path(self):
             with self.assertRaises(FileNotFoundError):
                 utils.find_folder_path('this/is/not/a/path', None)
 
+    def test11_is_ndarray_convertable(self):
+        '''Test of the function utils.is_ndarray_convertable'''
+        class NumpyLike:
+            def __init__(self, *args):
+                self.args = args
+                self.dtype = None
+            def astype(self, _):
+                return self
+            def tolist(self):
+                return self.args
+        
+        for obj, expected_result in (
+            (None, False),
+            ("some text", False),
+            (np.array(1), True),
+            (np.array(1.1), True),
+            (np.array([1, 2, 3], dtype=np.int64), True),
+            (np.array([[1], [2], [3]], dtype=np.int64), True),
+            (NumpyLike(1, 2, 3), True),
+        ):
+            self.assertEqual(expected_result, utils.is_ndarray_convertable(obj))
+
+    def test12_ndarray_to_builtin_object(self):
+        '''Test of the function utils.ndarray_to_builtin_object'''
```

Comment:
Adapt tests if we make changes to function def (e.g. raise error if not convertible)